backend/pom.xml

```diff
@@ -46,6 +46,17 @@
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+            <version>2.9.2</version>
```

Comment:
Move version name into maven properties section and reuse it for those two dependencies